<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonetaForm</class>
 <widget class="QWidget" name="MonetaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
font-weight: bold;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QDockWidget" name="dockWidget">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>117</height>
       </size>
      </property>
      <property name="floating">
       <bool>false</bool>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="windowTitle">
       <string>Collezione</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="setupCollezione">
          <property name="text">
           <string>Collezione</string>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/icons/process_info.png</normaloff>:/icons/process_info.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="itemList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableView {
/* font-size: 8px;*/
font-family: sans;
selection-background-color: rgba(100, 100, 100, 255)
selection-color: black;
}

</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>19</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTabWidget" name="Tabs">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab {
	border-left: 1px solid gray;
	border-right: 1px solid gray;
	border-top: 1px solid gray;
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
	min-width: 20ex;
	padding: 10px;
	font-size: 15px;
	font-weight: bold;
	font-style: small-caps;
 }

QTabBar::tab:selected {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
 }</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMoneta">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Moneta</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="superiore" native="true">
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QWidget" name="sup_sin" native="true">
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <property name="horizontalSpacing">
               <number>3</number>
              </property>
              <property name="verticalSpacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Stato</string>
                </property>
                <property name="buddy">
                 <cstring>led</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLedIndicator" name="led">
                <property name="text">
                 <string>PushButton</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Id</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="id">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>id</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Posizione</string>
                </property>
                <property name="buddy">
                 <cstring>posizione</cstring>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QPushButton" name="posizione">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>PushButton</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>true</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="17" column="0" colspan="2">
               <widget class="DatiFisiciWidget" name="datiFisici">
                <property name="title">
                 <string>Dati Fisici</string>
                </property>
               </widget>
              </item>
              <item row="21" column="0" colspan="2">
               <widget class="DatiAcquistoWidget" name="datiAcquisto">
                <property name="title">
                 <string>Acquisto</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0" colspan="2">
               <widget class="EmissioneForm" name="emissione">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="title">
                 <string>Emissione</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QWidget" name="sup_des" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="LetteraturaForm" name="letteratura">
                <property name="title">
                 <string>Letteratura</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="NoteForm" name="note">
                <property name="title">
                 <string>Note</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Documenti legati alla moneta</string>
                </property>
                <property name="buddy">
                 <cstring>documenti</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="documenti">
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QWidget" name="sup_cen" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Ambiti</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QListView" name="ambiti">
                   <property name="alternatingRowColors">
                    <bool>true</bool>
                   </property>
                   <property name="selectionBehavior">
                    <enum>QAbstractItemView::SelectItems</enum>
                   </property>
                   <property name="wordWrap">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="ZeccaWidget" name="zeccaEzecchieri">
                <property name="title">
                 <string>Zecche e zecchieri</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="inferiore" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="DescrizioneForm" name="dritto">
             <property name="title">
              <string>Dritto</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DescrizioneForm" name="rovescio">
             <property name="title">
              <string>Rovescio</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DescrizioneForm" name="taglio">
             <property name="title">
              <string>Taglio</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Vassoi">
       <attribute name="title">
        <string>Vassoi</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QTabWidget" name="tabsVassoi"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLedIndicator</class>
   <extends>QPushButton</extends>
   <header>qledindicator.h</header>
  </customwidget>
  <customwidget>
   <class>DescrizioneForm</class>
   <extends>QGroupBox</extends>
   <header>descrizioneform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatiAcquistoWidget</class>
   <extends>QGroupBox</extends>
   <header>datiacquistowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatiFisiciWidget</class>
   <extends>QGroupBox</extends>
   <header>datifisiciwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZeccaWidget</class>
   <extends>QGroupBox</extends>
   <header>zeccawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NoteForm</class>
   <extends>QGroupBox</extends>
   <header>noteform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EmissioneForm</class>
   <extends>QGroupBox</extends>
   <header>emissioneform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LetteraturaForm</class>
   <extends>QGroupBox</extends>
   <header>letteraturaform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ambiti</tabstop>
  <tabstop>documenti</tabstop>
  <tabstop>setupCollezione</tabstop>
  <tabstop>posizione</tabstop>
  <tabstop>itemList</tabstop>
  <tabstop>led</tabstop>
  <tabstop>Tabs</tabstop>
  <tabstop>tabsVassoi</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
